polaris-react/src/components/SortIcon/SortIcon.scss

```diff
@@ -0,0 +1,24 @@
+@import '../../styles/common';
+
+.SortIcon {
+  height: var(--p-space-5);
+  width: var(--p-space-5);
+}
+.Icon {
+  position: relative;
+  pointer-events: none;
+  svg {
+    fill: var(--p-icon-subdued);
+  }
+  &:first-of-type {
+    top: -3.5px;
```

Comment:
Are these (this and the `-16.5px` below) relative to anything or are they adjustments made to make things look good?